spec/cases/each_with_ar_sqlite.rb

```diff
@@ -0,0 +1,47 @@
+require './spec/cases/helper'
+require "active_record"
+require "sqlite3"
+STDOUT.sync = true
+
+
+ActiveRecord::Schema.verbose = false
+ActiveRecord::Base.establish_connection(
+  :adapter => "sqlite3",
+  :database => ":memory:"
+)
+
+class User < ActiveRecord::Base
+end
+
+# create tables
+unless User.table_exists?
+  ActiveRecord::Schema.define(:version => 1) do
+    create_table :users do |t|
+      t.string :name
+    end
+  end
+end
+
+User.delete_all
+
+3.times { User.create!(:name => "X") }
+
+print "Parent: "
+puts User.first.name
+
+print "Parallel (fork): "
+Parallel.each(User.all, in_processes: 3) do |user|
```

Comment:
Oh man... You're right.  I guess SQLite isn't working.
I made some adjustments.  I'll have a new commit in a sec.... and a pending test. :(